### PR TITLE
Feature/vsp 1256

### PR DESCRIPTION
**Pull request by joecollier**
- Adding functionality to the formatter to handle a configuration value that determines where currency symbol is place. i.e. $9.99 or 9.40€

### CODE REVIEW

File: src/Currency/Formatter.php [hunk 1]
```diff
@@ -127,8 +130,12 @@ protected static function handleLabel(Config $config)
      *
      * @return string A formatted string
      */
-    protected static function handleFormat($symbol, $number, $label)
+    protected static function handleFormat($symbol, $number, $label, $prepend_symbol)
```
Comment (Nicole8164) [inline src/Currency/Formatter.php]: Please add this to the docblock

File: tests/spec/Currency/Formatter.spec.php [hunk 2]
```diff
@@ -61,6 +61,16 @@
             }
         });
 
+        it('formats a number into a currency string 2', function () {
```
Comment (Nicole8164) [inline tests/spec/Currency/Formatter.spec.php]: Can we update this to be more specific like `formats a number into a currency string with symbol after number` and wrap this it statement in a context like `context('when prepend_symbol is false', function())`?

Comment (joecollier) [inline tests/spec/Currency/Formatter.spec.php]: Thanks, I missed that one.

File: src/Currency/Formatter.php [hunk 3]
```diff
@@ -124,11 +127,16 @@ protected static function handleLabel(Config $config)
      * @param string $symbol
      * @param number $number
      * @param string $label
+     * @param string $prepend_symbol
```
Comment (Nicole8164) [inline src/Currency/Formatter.php]: Should this be a boolean?

File: src/Currency/Formatter.php [hunk 4]
```diff
@@ -124,11 +127,16 @@ protected static function handleLabel(Config $config)
      * @param string $symbol
      * @param number $number
      * @param string $label
+     * @param bool $prepend_symbol
      *
      * @return string A formatted string
      */
-    protected static function handleFormat($symbol, $number, $label)
+    protected static function handleFormat($symbol, $number, $label, $prepend_symbol)
```
Comment (angelxmoreno) [inline src/Currency/Formatter.php]:
```php
return  (!$prepend_symbol)
    ? sprintf('%s%s%s', $number, $symbol, $label)
    : sprintf('%s%s%s', $symbol, $number, $label);
```

Comment (joecollier) [inline src/Currency/Formatter.php]: Personal preference. (I like guard clauses)

File: src/Currency/Formatter.php (hunk 1 above)
Comment (angelxmoreno) [inline src/Currency/Formatter.php]: can we give `prepend_symbol` a default value of false ?

**Pull request merged**